ros2_batch_job/__main__.py

```diff
@@ -239,6 +239,15 @@ def get_args(sysargv=None):
     parser.add_argument(
         '--visual-studio-version', default=None, required=(os.name == 'nt'),
         help='select the Visual Studio version')
+    parser.add_argument(
+        '--source-space', default='src',
```

Comment:
Please use `dest='sourcespace'` to avoid having to renamed the used `args`. That will make the patch much more concise.

Same for the other two options.

Reply:
Done